server/hosts.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  reviews: 'http://localhost:4006'
```

Comment:
Did you do this so that when you are deployed you only have to change the forwarding url in one location? ...clever.

Reply:
Indeed.